main.go

```diff
@@ -37,5 +38,7 @@ func main() {
 		cmd.CmdAdmin,
 	}
 	app.Flags = append(app.Flags, []cli.Flag{}...)
-	app.Run(os.Args)
+	err := app.Run(os.Args)
```

Comment:
It's not using any logger, it's only printing to stdout. So we will have the same bug again